src/components/page-layout/NavigationBar.js

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
 
 // Componenets
-import { Button } from '@material-ui/core';
+import { Button, Grid } from '@material-ui/core';
 
 // Styles
 import styles from './styles/navigation-bar.module.css';
 
 const NavigationBar = () => (
-        <div className={styles.container}>
-            <Button className={styles.button}>Home</Button>
-            <Button className={styles.button}>Groceries</Button>
-            <Button className={styles.cart}>Cart</Button>
+  <div className={styles.container}>
+    <Grid container justify="space-between">
+      <Grid item>
+        <div>
+          <Button className={styles.button}>Home</Button>
+          <Button className={styles.button}>Groceries</Button>
         </div>
+      </Grid>
+
+      <Grid item>
+        <Button className={styles.cart}>Cart</Button>
```

Comment:
I'm sure Material-UI has built in SVG icons to change this to a cart super easily.

Imports:
```
import StorefrontIcon from '@material-ui/icons/Storefront';
import ShoppingBasketIcon from '@material-ui/icons/ShoppingBasket';
import HomeIcon from '@material-ui/icons/Home';
```

For Home And Groceries:
```
<Button className={styles.button}>
  <HomeIcon style={{ color: green }} />
</Button>
<Button className={styles.button}>
  <StorefrontIcon style={{ color: green }} />
</Button>
```

For Cart:
```
<Button className={styles.button}>
  <ShoppingBasketIcon style={{ color: green }} />
</Button>
```

Reply:
Also, I'm not quite sure why we need Home & Grocery nav links. Won't they lead to the same place?

Reply:
Sounds good.

As far as why we have the two buttons, they very well might end up leading to the same place but that's just what's on the wireframes right now so we can change them if that ends up being true